README.md

```diff
@@ -1,61 +1,63 @@
-# ArduinoIoTCloud
+`ArduinoIoTCloud`
+=================
 [![Build Status](https://travis-ci.org/arduino-libraries/ArduinoIoTCloud.svg?branch=master)](https://travis-ci.org/arduino-libraries/ArduinoIoTCloud)
 
-This library facilitates interactions between boards featuring a cryptography co-processor and the Arduino IoT Cloud service. It includes a ConnectionManager to handle connection/disconnection/reconnection flows and provides means to interface local sketch variables with cloud-based Thing properties, enabling synchronization and on-change callbacks.
-Currently supported boards: MKR1000 (WiFi101) and MKR 1010 (WiFiNINA). Support for MKRGSM is nearing completion and more cryptography enabled boards are following.
+### What?
+The `ArduinoIoTCloud` library is the central element of the firmware enabling certain Arduino boards to connect to the [Arduino IoT Cloud](https://create.arduino.cc/iot/). The following boards are supported:
```

Comment:
```suggestion
The `ArduinoIoTCloud` library is the central element of the firmware enabling certain Arduino boards to connect to the [Arduino IoT Cloud](https://www.arduino.cc/en/IoT/HomePage). The following boards are supported:
```
[This page](https://www.arduino.cc/en/IoT/HomePage) might be a better introduction for someone in the "what?" phase. The previous link requires logging in to an Arduino account to enter, which might result in a "well, I didn't want to know that badly" response.

Either one is fine; it's just a thought.